.travis.yml

```diff
@@ -50,8 +50,9 @@ jobs:
   include:
     - stage: normal builds
       os: linux
-      compiler: clang
```

Comment:
I think you can set `clang-4.0` here to achieve the same effect.

Reply:
I tried that, but cmake would always complain that the sanitizer can only be used with clang. I think the `compiler` setting just sets `$CC` which doesn't seem to be enough. But even with `CMAKE_C_COMPILER` set I get that message. ;x

Reply:
That's because you don't have _any_ compiler set, so it [gets set to gcc](https://travis-ci.org/neovim/neovim/jobs/457719820#L798-L799).

Reply:
Oh, I see. My `$CC` got overruled by the default `compiler`.